lms/djangoapps/courseware/views.py

```diff
@@ -701,12 +701,11 @@ def course_info(request, course_id):
         }
 
         # Get the URL of the user's last position in order to display the 'where you were last' message
-        context['last_accessed_courseware'] = None
+        context['last_accessed_courseware_url'] = None
         if SelfPacedConfiguration.current().enable_course_home_improvements:
-            (section_module, section_url) = get_last_accessed_courseware(course, request)
-            if section_module is not None and section_url is not None:
-                context['last_accessed_courseware'] = section_module
-                context['last_accessed_url'] = section_url
+            section_url = get_last_accessed_courseware(course, request)
+            if section_url is not None:
```

Comment:
is this check necessary? Seems like we can assign even if it is `None` and simplify this to `context['last_accessed_courseware_url'] = get_last_accessed_courseware(course, request)`